db/import.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+import fs from 'fs';
+import createDBConnectionPool from '../server/db';
+
+const BATCH_SIZE = 1000;
+
+const data = fs.readFileSync(process.argv[2], 'utf8')
+  .split('\n')
+  .filter(Boolean)
+  .map((row) => {
+    const rowData = row.split(',');
```

Comment:
Not really a biggie that's important but could you extract the following:
```
const rowData = row.split(',');
rowData[0].slice(1, -1)
```

into a function?